geomesa-index-api/src/main/scala/org/locationtech/geomesa/index/utils/KryoLazyDensityUtils.scala

```diff
@@ -176,4 +176,4 @@ object KryoLazyDensityUtils {
       }
     }
   }
-}
+}
```

Comment:
can you revert this change to keep the newline at the end of the file? we try to do that for @tkunicki 's sake :)